### ISSUE TITLE
感谢开源，代码有一点点问题，这里指出，供参考。

### ISSUE THREAD

**Issue by foboshi** [moon-hotel/TransformerCouplet#3]
train.py的114行应该改成              config.writer.add_scalar('Training/Loss', loss.item(), learning_rate.step)

**Comment by mlwithme** [comment]
多谢，已修正。
